client/providers/CallProvider/hooks/useVoipClient.ts

```diff
@@ -21,50 +21,46 @@ export const isUseVoipClientResultError = (result: UseVoipClientResult): result
 	!!(result as UseVoipClientResultError).error;
 
 export const isUseVoipClientResultLoading = (result: UseVoipClientResult): result is UseVoipClientResultLoading =>
-	Object.keys(result).length === 0;
+	result === undefined || Object.keys(result).length === 0;
```

Comment:
```suggestion
	!result || !Object.keys(result).length;
```

Both undefined and 0 are falsy values, so using the ! the code looks cleaner

Reply:
fixed